test/datadog-operator/operator_deployment_test.go

```diff
@@ -56,6 +56,18 @@ func Test_operator_chart(t *testing.T) {
 			assertions: verifyAll,
 			skipTest:   SkipTest,
 		},
+		{
+			name: "livenessProbe is correctly configured",
```

Comment:
This one tests just default case. Consider adding one to verify overrides since this is the feature being added, e.g. 
```go
				Overrides: map[string]string{
					"livenessProbe.timeoutSeconds":    "99",
					"livenessProbe.periodSeconds":    "99",
					"livenessProbe.failureThreshold": "9",
					// etc
				},
```